scripts/add_electricity.py

```diff
@@ -302,14 +304,13 @@ def attach_conventional_generators(
                     n.generators.loc[idx].bus.map(bus_values).dropna()
                 )
             else:
-                # Single value affecting all generators of technology k indiscriminantely of country
+                # Single value affecting all generators of technology k indiscriminately of country
                 n.generators.loc[idx, attr] = values
 
 
-def attach_storageunits(n, costs, technologies, extendable_carriers):
+def attach_storageunits(n, costs, number_microgrids, technologies, extendable_carriers):
```

Comment:
See comment above: it is better to pass as argument the list of buses where this storage shall be added or we need a flag that is automatically parsed here, but I would avoid having the bus names hard-coded here.